wear/src/main/java/io/homeassistant/companion/android/home/views/SelectShortcutsTileView.kt

```diff
@@ -0,0 +1,119 @@
+import com.mikepenz.iconics.compose.Image
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.theme.WearAppTheme
+import io.homeassistant.companion.android.theme.wearColorPalette
+import io.homeassistant.companion.android.views.ListHeader
+import io.homeassistant.companion.android.views.ThemeLazyColumn
+import io.homeassistant.companion.android.common.R as commonR
+
+@Composable
+fun SelectShortcutsTileView(
+    shortcutTileEntitiesCountById: Map<Int?, Int>,
+    onSelectShortcutsTile: (tileId: Int?) -> Unit,
+    isShowShortcutTextEnabled: Boolean,
+    onShowShortcutTextEnabled: (Boolean) -> Unit
+) {
+    val scalingLazyListState = rememberScalingLazyListState()
+    WearAppTheme {
+        Scaffold(
+            positionIndicator = {
+                if (scalingLazyListState.isScrollInProgress) {
+                    PositionIndicator(scalingLazyListState = scalingLazyListState)
+                }
+            },
+            timeText = { TimeText(scalingLazyListState = scalingLazyListState) }
+        ) {
+            ThemeLazyColumn(state = scalingLazyListState) {
+                item {
+                    ListHeader(id = commonR.string.shortcut_tiles)
+                }
+                item {
+                    ToggleChip(
+                        modifier = Modifier.fillMaxWidth(),
+                        checked = isShowShortcutTextEnabled,
+                        onCheckedChange = { onShowShortcutTextEnabled(it) },
+                        label = {
+                            Text(stringResource(commonR.string.shortcuts_tile_text_setting))
+                        },
+                        appIcon = {
+                            Image(
+                                asset =
+                                if (isShowShortcutTextEnabled) {
+                                    CommunityMaterial.Icon.cmd_alphabetical
+                                } else {
+                                    CommunityMaterial.Icon.cmd_alphabetical_off
+                                },
+                                colorFilter = ColorFilter.tint(wearColorPalette.onSurface)
+                            )
+                        },
+                        toggleControl = {
+                            Icon(
+                                imageVector = ToggleChipDefaults.checkboxIcon(isShowShortcutTextEnabled),
+                                contentDescription = if (isShowShortcutTextEnabled) {
+                                    stringResource(commonR.string.show)
+                                } else {
+                                    stringResource(commonR.string.hide)
+                                }
+                            )
+                        }
+                    )
+                }
+                item {
+                    ListHeader(id = commonR.string.shortcuts_tile_select)
```

Comment:
Maybe hide this title if there are no shortcut tiles, or add an empty state message telling to add a tile first?

Reply:
Done in e629142f